### PR TITLE
Mobile html and css Pull

### PR DESCRIPTION
**Pull request by SSEKA**
The first pull request after integration of HTML and CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@100;300;500;700&display=swap">
+</head>
+<body>
+    <div class="head-section">
+        <header>
+         <nav>
+            <a class="logo" href="#">My logo</a>
```
Comment (mahbubul14) [inline index.html]: - [x] You should move this `a` tag outside of the `nav` tag. :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/git.png" alt="github"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/peace.png" alt="peace"></a>
+           </li>
+        </ul>
+      </div>  
+    </div>
```
Comment (mahbubul14) [inline index.html]: - [x] Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly fix the background color. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
your | png

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @SSEKA,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] Your `README` file is not updated according to the project. Kindly work on it and make it looks professional.

- [x] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Qoosim) [inline index.html]: - [ ] As requested by the previous reviewer, kindly move `a` tag outside the `nav` tag.

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@
 
 
 
-- [Live Demo Link]([https://yourdeployedapplicationlink.com](https://github.com/SSEKA/Portolio.git))
+- [Live Demo Link]([https://yourdeployedapplicationlink.com](https://github.com/SSEKA/My-Portfolio.git))
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Qoosim) [inline README.md]: - [ ] As suggested by the previous reviewer, all the headings on the readme template must be present on your readme file except the FAQ section which is optional. At this moment, some headings are missing from your readme. Such as `Key Features and Future Features` etc. Kindly follow [Microverse template](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to make your professional readme.

File: index.html (hunk 2 above)
Comment (Qoosim) [inline index.html]: - [ ] As mentioned by the previous reviewer, Kindly ensure that your design looks exactly like that of figma design. At this moment, your design is not in line with what we have on the figma design. The Figma Design has white background, but your design does not have the same background and many other differences. Kindly give it a closer look and work your design alongside the figma design.

### Your Design
your-page

### Figma Design
temp-page

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @SSEKA, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <a href="#"><img class="social-icons" src="./images/more.png" alt="more"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/git.png" alt="github"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/peace.png" alt="peace"></a>
+           </li>
+        </ul>
+      </div>
```
Comment (Lizdev-05) [inline index.html]: I think you should be able to achieve the  design by adding a section that will be a wrapper for this bio part, you can give it a class name "main-container" and give it this style 

```
.main-container {
  width: 100%;
  height: 100%;
  z-index: 999;
}
```
Your design presently is not aligned with the Figma design... 😞 

| Your Design | Figma Design |
| :---:   | :---: | 
| image|image|

File: style.css [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+.bio {
+  /* margin: 0px 0; */
+  background-image: url(./images/mobile-background-img.png);
+  background-size: 100% 95%;
+  background-repeat: no-repeat;
+  padding: 10px;
+  padding-bottom: 118px;
+  border-bottom-left-radius: 15rem;
+}
```
Comment (Lizdev-05) [inline style.css]: Great job making the required changes 👍 . As requested by the previous reviewers, your project design is not in line with the Figma design.

I think you should consider using this styling...

 ```
 width: 100%;
  height: calc(718px - 60px);
 background-image: url(./images/mobile-background-img.png);
  background-position: center;
  background-size: contain;
  background-repeat: no-repeat;
  overflow: hidden;
  border-bottom-left-radius: 5rem;
  background-color: #fff;
  position: relative;
```

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @SSEKA, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights

- [ ] No linter errors. ✔️
- [ ] Correct GitHub Flow. ✔️
- [x] Descriptive README


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @SSEKA, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct GitHub Flow 👍

- All linters passed 🏅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**